com.unity.ml-agents.extensions/Runtime/Sensors/PhysicsSensorSettings.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+
+using Unity.MLAgents.Sensors;
+
+namespace Unity.MLAgents.Extensions.Sensors
+{
+    [Serializable]
+    public struct PhysicsSensorSettings
```

Comment:
This might get extended later to have different representation types for rotations, e.g. 3x3 rotation matrices, Euler angles.